test/ping/test_json.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
```

Comment:
FYI I plan to add tests similar to perl `Test` based tests like https://github.com/schweikert/fping/tree/develop/ci (e.g. https://github.com/schweikert/fping/blob/develop/ci/test-01-basics.pl).
Although I don't like perl, using well defined testsuite helps to concentrate on tests.
It's ok to have this, but just warn you that it will be replaced.

Reply:
Sounds good, happy to replace it when we have a common framework to handle these tests!